quarkus-build-caching-extension/pom.xml

```diff
@@ -143,6 +143,17 @@
                 </executions>
             </plugin>
         </plugins>
+        <resources>
+            <resource>
+                <directory>${project.basedir}/src/main/resources</directory>
+            </resource>
+            <resource>
```

Comment:
I would take this (src/main/resources) out here, too.

Reply:
As discussed in the CCUD Maven PR, we will leave it in.